### ISSUE TITLE
gatsby-image triggers lots of image GET request when resizing

### ISSUE THREAD

**Issue by davidbasalla** [gatsbyjs/gatsby#13911]
## Relevant information

I noticed that when resizing the browser, a "gatsby-image" `<Img />` component is continually making GET requests to fetch the image (multiple requests a second).

I first noticed this when I started using "gatsby-image" on my own project, and assumed that I was doing something wrong. However I also noticed it on the official (?) "gatsby-image" demo (https://using-gatsby-image.gatsbyjs.org/). When resizing the browser, the same image is fetched over and over again, and you can quickly rack up several megabytes of download usage just by continuing to resize the browser.

![Screen Shot 2019-05-07 at 17 16 42](https://user-images.githubusercontent.com/5112255/57315651-ed28a480-70eb-11e9-8f74-1a24e1426209.png)

Is that intentional?

I'm on OSX v10.12.6, Chrome v74

**Comment by Kevin-Hamilton** [comment]
Could you have a browser plugin causing this?  Have you tried in another browser?  I'm unable to reproduce this issue on https://using-gatsby-image.gatsbyjs.org/ in Chrome, Firefox or Safari.

**Comment by davidbasalla** [comment]
Thanks for checking, it looks like the culprit was actually the 'Disable cache' check box in the Chrome dev tools - it was checked. When I uncheck it, I don't get the multiple image requests, so I think everything is fine. I think I was misled by the fact that when I checked other websites with images (eg Wikipedia) I didn't see the multiple requests, even when disabling the cache.

I'll close this ticket!

**Comment by Kevin-Hamilton** [comment]
@davidbasalla I had tested with 'Disable cache' checked as well because I had wondered if that could be a culprit.  However, I was testing in Chrome 73.  I just updated to Chrome 74 and now I'm seeing the same as you.  I also tried Canary, and I see the same happening there as well.  

I've been able to now reproduce this on several other websites built in several other platforms.  So not a Gatsby issue. It appears to me to be a Chrome bug associated with img srcset and the "disable cache" flag in developer tools.  

Seems to me like it is probably a new variant of this old Chromium issue: https://bugs.chromium.org/p/chromium/issues/detail?id=625797
